server/routes/viewingIncidents/viewIncidents.ts

```diff
@@ -105,7 +108,15 @@ export default class ViewIncidentsRoutes {
         statements: submittedStatements,
         reportEditOrDeletePermitted,
       }
```

Comment:
prevent access to a reports edit history to users who are not the report owner or a reviewer/coordinator